app/assets/javascripts/student_profile/services_list.jsx

```diff
@@ -252,9 +252,20 @@ import _ from 'lodash';
       const isHovering = (this.state.hoveringDiscontinueServiceId === service.id);
       const isPending = (this.props.discontinueServiceRequests[service.id] === 'pending');
 
-      const buttonText = (isPending)
-        ? 'Updating...'
-        : (isConfirming) ? 'Confirm' : 'Discontinue Early';
+      var Text = 'Confirm';
+
+      if (isPending) {
```

Comment:
Button text was already a nested ternary operator. I think writing out the statements is more readable than an even more nested ternary operator. Also `const buttontext` is read only so I added a `text` variable to be modified then assigned to `const buttontext`.